receiver/awscontainerinsightreceiver/internal/neuron/pod_attribute_decorator_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package neuron
+
+import (
+	"context"
+	"testing"
+
+	ci "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/containerinsight"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/prometheusscraper/decoratorconsumer"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awscontainerinsightreceiver/internal/stores"
+	"go.opentelemetry.io/collector/consumer/consumertest"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+var dummyPodName = "pod-name"
+var dummyContainerName = "container-name"
+var dummyNamespace = "namespace"
+
+type mockPodResourcesStore struct {
+}
+
+func (m mockPodResourcesStore) GetContainerInfo(deviceIndex string, resourceName string) *stores.ContainerInfo {
```

Comment:
Need test for falling back to `neuronDeviceResourceNameAlt` if `neuronDeviceResourceName` is not found.

Reply:
Added )